Libs/Widgets/ctkDoubleSpinBox.cpp

```diff
@@ -398,13 +398,22 @@ ::validateAndInterpret(QString &input, int &pos,
   // could be because of group separators:
   if (!ok && state == QValidator::Acceptable)
     {
-    if (q->locale().groupSeparator().isPrint())
+#if QT_VERSION >= 0x060000
+    QChar groupSeparator;
+    if ( q->locale().groupSeparator().size() == 1 )
+        groupSeparator = q->locale().groupSeparator()[0];
+    else
+        assert( false );
```

Comment:
This would crash the application in debug mode, so please remove it. We can log a warning instead, but probably it is not needed. We could describe in a comment that the group separator does not necessarily fit into a QChar (https://bugreports.qt.io/browse/QTBUG-69324) but CTK only support group separators if they fit into a QChar. Probably we should add a comment about this into the ctkDoubleSpinBox class documentation.